sdk/servicebus/service-bus/src/core/linkEntity.ts

```diff
@@ -386,6 +393,7 @@ export abstract class LinkEntity<LinkT extends Receiver | AwaitableSender | Requ
     );
     await defaultLock.acquire(this._context.cbsSession.cbsLock, async () => {
       this.checkIfConnectionReady();
+      // TODO: Pass operationOptions to init() - core-amqp
```

Comment:
This can actually be done now that the change in core-amqp has been merged into master :smile:

Reply:
Done.